Document guidelines for writing release notes (#685)
anki-geo/ultimate-geography: CONTRIBUTING.md

- news on the project, future developments, etc.

The goal is to avoid users: feeling confused when discovering content changes as they review; missing some subtle changes (e.g. spelling, new supplementary information, etc.); or not finding out about changes for a long time because they happened in cards that they won't see again for a while.

We also don't want users who are interested in staying up to date with changes to have to subscribe to PRs or hunt for changes in git diffs.

With these goals in mind, it's important to write release notes that are **concise** yet **precise**, **well structured**, and **easy to read**. Keeping the structure, wording, emojis, etc. **consistent** across releases is also important and can be achieved by looking at and getting inspiration from past release notes.

The recommended structure, which can of course be tweaked and expanded as needed, is as follows:

1. Short introduction to summarise the most significant changes and/or give updates on the projet _(optional)_.
2. Concise upgrade instructions, typically pointing to detailled procedures in the README and wiki.
3. New translations, if any (including language code).

> 4. New notes, if any, with links to Wikipedia and optionnally justification on why those notes were added (e.g. new/changed inclusion criteria)

Reviewer: ```suggestion
4. New notes, if any, with links to Wikipedia and optionally justification on why those notes were added (e.g. new/changed inclusion criteria)
```

Typo (optionally).

5. Changes to country names, capitals and supplementary info.
6. Changes to maps and flags.
7. Other changes, for instance to the structure of the deck (tags, description, etc.), the documentation (when significant), or the BrainBrew recipes and development tool chain (if relevant to power users).
8. Contributors (at least new contributors) _(optional)_

Here are som additional guidelines:

> - If the upgrade is non-trivial, make sure to grab users attention with bold text, emojis, etc. at the top of the release notes.

Reviewer: ```suggestion
- If the upgrade is non-trivial, make sure to grab users' attention with bold text, emojis, etc. at the top of the release notes.
```

> - Don't write release notes from a contributor's point of view (i.e. one bullet point per pull request/commit/issue), but from a users point of view (i.e. one bullet point per meaningful change). Indeed, it's common for one change to span multiple pull requests (e.g. new translation followed by various corrections), and it sometimes happen that multiple changes are done with a single pull request.

Reviewer: ```suggestion
- Don't write release notes from a contributor's point of view (i.e. one bullet point per pull request/commit/issue), but from a user's point of view (i.e. one bullet point per meaningful change). Indeed, it's common for one change to span multiple pull requests (e.g. new translation followed by various corrections), and it sometimes happens that multiple changes are done with a single pull request.
```

> - Be as specific as possible, for instance by mentioning the country name of the affected note, except when a lot of notes are affected by similar minor changes (like when a lot typos/mistranslations are fixed in a specific language).

Reviewer: ```suggestion
- Be as specific as possible, for instance by mentioning the country name of the affected note, except when a lot of notes are affected by similar minor changes (like when a lot of typos/mistranslations are fixed in a specific language).
```
